include/deal.II/grid/grid_tools.h

```diff
@@ -643,6 +643,17 @@ namespace GridTools
   std::vector<std::set<typename Triangulation<dim,spacedim>::active_cell_iterator> >
   vertex_to_cell_map(const Triangulation<dim,spacedim> &triangulation);
 
+  /**
+   * Compute a global unique index for each vertex and hanging node associated
+   * to a locally owned active cell. The key of the map is the local index of
```

Comment:
What I am trying to say is that the vertices of a ghost cell that are hanging nodes of a locally owned cells have a global index. However, the other vertices of the ghost cells that do not "touch" an active cell do not have a global index on this processor.